src/app/public/modules/modal/modal.component.spec.ts

```diff
@@ -123,17 +135,12 @@ describe('Modal component', () => {
       ]
     });
 
-    applicationRef = TestBed.inject(ApplicationRef);
```

Comment:
I had to refactor these tests to inject these classes on-demand instead of immediately after `TestBed.configureTestingModule()`, since a test needed to be able to call `TestBed.overrideProvider()` which can't be called after `TestBed.inject()` is called.